CONTRIBUTING.md

```diff
@@ -196,3 +196,39 @@ In this example this can be done by invoking following command from the reposito
 ```bash
 ./gradlew test -Pcoordinates=org.example:library:0.0.1
 ```
+
+## Tested Libraries and Frameworks
+
+If your library or framework is tested with GraalVM Native Image, consider adding it to [this list](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list.json).
+
+Write an entry as follows:
+```json
+{
+    "artifact": "<groupId>:<artifactId>",
+    "description": "<artifactDescription>",
+    "details": [
+      {
+        "minimum_version": "<minimumVersion>",
+        "maximum_version": "<maximumVersion>",
+        "metadata_locations": ["<metadataLocations>"],
+        "tests_locations": ["<testLocations>"],
+        "test_level": "<testLevel>"
+      }
+    ]
+}
+```
+Where:
+ * `<groupId>` and `<artifactId>` - standard Maven coordinates ([see this](https://maven.apache.org/pom.html#maven-coordinates))
```

Comment:
Maven coordinates include `<version>`, which are not part of this. I think it's fine to just say "`artifact` is the name in the groupId:artifactId format".